extensions/create-t3-app/src/index.tsx

```diff
@@ -0,0 +1,174 @@
+  git: boolean;
+  install: boolean;
+};
+
+export default function Command(props: LaunchProps<{ arguments: Arguments }>) {
+  const prefs = getPreferenceValues<Preferences>();
+  const [nameError, setNameError] = useState<string | undefined>();
+  const [langError, setLangError] = useState<string | undefined>();
+
+  function dropNameErrorIfNeeded() {
+    if (nameError && nameError.length > 0) {
+      setNameError(undefined);
+    }
+  }
+
+  async function handleSubmit(values: Values) {
+    // in case the user updates their preferences while the form is open
+    const prefs = getPreferenceValues<Preferences>();
+
+    const folder = values.dir[0];
+    if (!fs.existsSync(folder) || !fs.lstatSync(folder).isDirectory()) {
+      return false;
+    }
+
+    const command = [getCreateCommand(prefs.packageManager)];
+
+    command.push(values.appName || "my-t3-app");
+
+    if (!values.git) command.push("--noGit");
+    if (!values.install) command.push("--noInstall");
+
+    command.push("--CI");
+
+    if (values["next-auth"]) command.push("--nextAuth");
+    if (values.prisma) command.push("--prisma");
+    if (values.tailwind) command.push("--tailwind");
+    if (values.trpc) command.push("--trpc");
+
+    console.log(command.join(" "));
+
+    showToast({
+      title: "Creating T3 app...",
+      message: "Check the terminal window to see how it's going",
+    });
+
+    await runAppleScript(`
+      tell application "Terminal"
+        activate
+        set shell to do script "cd ${folder} && ${command.join(" ")}"
+      end tell
+    `);
+  }
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm onSubmit={handleSubmit} title="Create app (opens Terminal)" />
+          <Action onAction={() => openCommandPreferences()} title="Change extension preferences" />
+        </ActionPanel>
+      }
+    >
+      <Form.TextField
+        id="appName"
+        title="Project name"
+        placeholder="my-t3-app"
+        defaultValue={props.arguments.appName}
+        error={nameError}
+        onChange={(value) => {
+          dropNameErrorIfNeeded();
+          if (!value) return;
+          if (!/^[\w-]+$/.test(value)) {
+            setNameError("Project name cannot contain spaces or special characters");
+          }
+        }}
+        autoFocus
+      />
+      <Form.FilePicker
+        id="dir"
+        title="Location"
+        canChooseDirectories
+        canChooseFiles={false}
+        allowMultipleSelection={false}
+        defaultValue={[prefs.directory]}
+      />
+      <Form.Dropdown
+        id="language"
+        storeValue
+        title="JS or TS?"
+        defaultValue="ts"
+        error={langError}
+        onChange={(value) => {
+          switch (value) {
+            case "ts":
+              setLangError(undefined);
+              break;
+            case "js":
+              setLangError("Wrong answer! Pick TypeScript!");
+              break;
+          }
+        }}
+      >
```

Comment:
May I ask what is the purpose of this? Is it humour or unsupported feature? Why have the option at all? 🤔

Reply:
Humor I guess? The CLI gives you the option to pick a language but forces you to pick JavaScript - I thought I would include it  in the raycast version to be faithful to that

Reply:
Agree, let's just remove it 😊